README.md

```diff
@@ -280,6 +280,49 @@ Please go the Remote Management tab of the Settings page and check which remote
 
 ![Remote Management Tab](./assets/screenshot-3.png)
 
+### RESTapi for Remote Management
```

Comment:
Seems that here: https://developer.wordpress.com/docs/api/ and in other places they use REST API, so I changed it into that.